addons/lunch/data/lunch_demo.xml

```diff
@@ -195,8 +199,17 @@
         </record>
 
         <record model="lunch.alert" id="alert_1">
-            <field name="message">Lunch must be ordered before 10h30 am</field>
+            <field name="alert_type">week</field>
+            <field name="monday">t</field>
```

Comment:
This works only by accident because Python will cast the string `t` to `True` (try `bool('f')` in a Python console).

If you want to insert booleans as (demo) data, use an `eval`, eg:
```xml
<field name="monday" eval="True" />
<field name="sunday" eval="False" />
```